content/doc/book/security/controller-isolation.adoc

```diff
@@ -69,7 +138,20 @@ In Jenkins 2.325 and earlier, it is enabled by default, but can be disabled in t
 
 IMPORTANT: It is strongly recommended that you not disable the Agent &rarr; Controller Access Control system.
 
-image::security/configure-global-security-agent-controller-toggle.png["Configure Global Security - Enable Agent => Controller Access Control", role=center]
-
 As an alternative to disabling Agent &rarr; Controller Access Control, in Jenkins 2.325 and earlier, administrators can selectively allow greater access.
 See link:/doc/book/security/controller-isolation/agent-to-controller/[the documentation] for details.
+
+=== Other Tips and Tricks
+
+Another way to prevent builds from running on the controller
+is to use a plugin such as the plugin:job-restrictions[Job Restrictions Plugin]
+to limit which jobs can be run on certain nodes (like the built-in controller node),
+independent of what your less trusted users may use as label expression in their job configurations.
+
+If you do not have any other computers on which to run agents,
+you can run an agent process as a different operating system user on the same system
+to achieve a similar isolation effect.
+In this case,
+be sure that the agent process has neither read nor write file system access
+to the Jenkins home directory,
+and that the agent process cannot use `sudo` or other practices to elevate its own permissions.
```

Comment:
At this point, with Docker, VMs, EC2 and similar cloud plugins, etc., I do wonder whether this is even relevant advice anymore. It's a neat trick for niche situations, but is it any more than that?